components/Footer/Footer.tsx

```diff
@@ -10,6 +10,10 @@ import UpRightArrowBlack from "public/assets/up-right-arrow-black.svg";
 import { VoteTicker } from "components";
 import { QUERIES, BREAKPOINTS } from "constants/breakpoints";
 import { useWindowSize } from "hooks";
+import MailchimpSubscribe from "react-mailchimp-subscribe";
+
+const MAILCHIMP_URL =
```

Comment:
this shoudl eventually be an env

Reply:
@daywiss agreed. Someone needs to check what's going on as the current production marketing website seems to create a 404. I need the updated URL.